### PR TITLE
add Agoric to README

### PR DESCRIPTION
**Pull request by tatyana0219**
Adds the following events or companies:
 - Agoric
 - https://www.eventbrite.com/e/postponed-rethinking-distributed-programming-tickets-97404019043

### Checklist

 - [X] I have updated the event or company counts
 - [X] I have put the most recent relevant date in the "Last Update" column
 - [X] I have linked to the article about the change, not the company's website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
 <a name="companies"></a>
-## Companies - 149
+## Companies - 150
 
 | Company | WFH | Travel | Visitors | Events | Last Update |
 | --- | --- | --- | --- | --- | --- |
 | Aaptiv | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
 | ActionIQ | Encouraged | Restricted | Restricted | Restricted | 2020-03-09 |
 | Adobe[[1]](https://www.npr.org/2020/03/03/811728989/coronavirus-cancellations-and-travel-bans-google-is-latest)[[2]](https://twitter.com/AdobeNews/status/1236758404330389504) | Encouraged / Directed for Seattle and Bay Area till 03/23/20 | Restricted | ? | ? | 2020-03-09 |
+| Agoric | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
```
Comment (drewbrew) [inline README.md]:
```suggestion
| [Agoric](https://www.eventbrite.com/e/postponed-rethinking-distributed-programming-tickets-97404019043) | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
```

**Comment by drewbrew** [comment]
Hi, this looks good. Is there a public post on the company policy by chance?

**Comment by dtribble** [comment]
Not yet. We changed modes at the beginning of the week, but haven't posted a public document. We will update when we do. (Dean Tribble - CEO).

**Comment by drewbrew** [comment]
Thanks!

**Comment by drewbrew** [comment]
Unfortunately, GitHub won't let me resolve the merge conflict (company counts). Once that's fixed, I'll merge.

**Comment by tatyana0219** [comment]
@drewbrew this is updated. Thank you.

**Comment by phildini** [comment]
Hi there! We recently re-worked the site to use yaml data files. We'd be happy to accept your PR once it's been reworked to use the right format.

Thanks!

**Comment by tatyana0219** [comment]
@phildini  all done thank you :)

**Comment by dtribble** [comment]
A small follow-up; our discussion of company policy: https://medium.com/agoric/11-things-covid-19-taught-us-about-the-decentralized-workplace-c03e1efebdc5

**Pull request merged**